tests/api.c

```diff
@@ -21752,14 +21752,14 @@ static void test_wolfSSL_ASN1_TIME_adj(void)
     /* offset_sec = -45 * min;*/
     asn_time = wolfSSL_ASN1_TIME_adj(s, t, offset_day, offset_sec);
     AssertTrue(asn_time->type == asn_utc_time);
-    XSTRNCPY(date_str, (const char*)&asn_time->data, sizeof(date_str));
+    XSTRNCPY(date_str, (const char*)&asn_time->data, sizeof(date_str)-1);
```

Comment:
This can still allow a non-null terminated date_str. To ensure it is make sure last byte of date_str is zero.

Reply:
Check